radicle-httpd/src/api/v1/projects.rs

```diff
@@ -360,14 +362,49 @@ async fn issue_handler(
         "id": issue_id,
         "author": issue.author(),
         "title": issue.title(),
-        "status": issue.status(),
-        "discussion": issue.comments().collect::<Vec<_>>(),
+        "state": issue.status(),
+        "discussion": issue.comments().collect::<Comments>(),
         "tags": issue.tags().collect::<Vec<_>>(),
     });
 
     Ok::<_, Error>(Json(issue))
 }
 
+#[derive(Serialize)]
+struct Author {
+    id: PublicKey,
+}
+
+#[derive(Serialize)]
+struct Comment {
+    author: Author,
+    body: String,
+    reactions: [String; 0],
+    timestamp: Timestamp,
+    reply_to: Option<PublicKey>,
```

Comment:
This should be `Option<CommentId>`, but we still need to define what `CommentId` should be on the frontend. Perhaps just an index into the discussion list..

Reply:
fixed [here](https://github.com/radicle-dev/heartwood/compare/9cb50027918aee4a79a93b35645c0f6dbd698ddd..55bfe8f4b7fda0a9208b59f4cf45439ea8b44f1e#diff-85a73839d407aa508b9e78b3756b0e9b6e8d1087a0cafac6a8325a8cf427b65eR385)